### DISCUSSION TITLE
GKUI19 firmware installation

### DISCUSSION THREAD

**Discussion by akg996** [xeon1989/Proton-X50-APK-Installer#37] [General]
Hi Xeonn, been a huge fan of ur work👌🏻

I wanna try downgrading to gkui19 from atlas
But i do not have the file, could u kindly share it?
Much appreciated~~
Im x50fs btw ya

**Comment by xeon1989** [comment]
Hi, mind to share your Hardware version in Settings - System?

**Reply by akg996** [reply]
![16891575901896913119197930666420](https://github.com/xeon1989/Proton-X50-APK-Installer-GKUI/assets/139234986/4215eead-b076-416f-ad89-8199f038974d)

Of coursee my man there ya go~

**Comment by xeon1989** [comment]
Here you go.
GKUI firmware.
https://mega.nz/file/TF0GhDjS#Q1cvUBjk-nxwBuTUgbWWKuyJX5r6gBrXDq8KYQURI5E

Download and place this file in FAT32 formatted USB.
Rest of steps should be the same as mentioned here.
https://github.com/xeon1989/Proton-X50-IHU-Upgrade-Downgrade

**Reply by akg996** [reply]
You're the mann
But if im not mistaken after unzipping the file
I have to place those files into a folder and rename it into something like .sh for the ihu to recognise the usb upgrade. 
Do u know what it is?

**Reply by xeon1989** [reply]
no meed. just download and place the file in USB, follow steps in guide to "update".

**Reply by akg996** [reply]
Yes my mann it worked flawlessly👌🏻
We go out makan my treat haha 
Where do ya live?

**Reply by RadenAmzar** [reply]
@xeon1989 hi, is there any GKUI firmware file yet for executive/standard?

**Reply by xeon1989** [reply]
@RadenAmzar No.

**Comment by Varto1u** [comment]
Hello, I do all the steps but it doesn't work with my geely coolray , here is the version ![image](https://github.com/xeon1989/Proton-X50-APK-Installer-GKUI/assets/124306538/5b1a4e02-0251-4305-a3de-2985c15b6eaf)

**Reply by xeon1989** [reply]
Hi, do NOT attempt to flash Proton's GKUI firmware for any other car than Proton X50. 
It is a sure way to break it!

**Reply by Varto1u** [reply]
> Hi, do NOT attempt to flash Proton's GKUI firmware for any other car than Proton X50. 
> 
> It is a sure way to break it!

So how to install ES Explorer on my GEELY Coolray, I tried all methods but not work, Could you please help me?

**Reply by xeon1989** [reply]
Unfortunately I do not have a solution for your Coolray.
As each firmware version implement things differently, without access to your headunit or firmware file to perform analysis, it is pretty much impossible for me to tell the best method works for your car.

**Comment by JayChong12** [comment]
Hi. Got any solution atlas version downgrade to gkui firmware? Proton X50 Executive

**Reply by xeon1989** [reply]
Hi, no for now.

**Reply by JayChong12** [reply]
If your sharing the 379 version? Only can use at Premium?

**Comment by Msh36345** [comment]
Hi,
I have a 2024 facelift model of the Geely Geometry C, also known in some regions as the Geometry M6.
The infotainment system looks identical to the one in the Geely Coolray.
I have a system update file for the Coolray that adds Apple CarPlay, but I’m concerned it might damage my current system. Unfortunately, I don’t have the original installation file for the current version of my system in case something goes wrong.

Is there a way to obtain the original firmware for my version?
Or is there another way to install apps on my car’s system?

Thank you!

**Reply by xeon1989** [reply]
Hi Shulman, sorry but I do not know the way to obtain original firmware for Geometry C/M6. 
Whether it might damage your current system, it really depends on what your system update files does to the HU. 

In Malaysia, we are bound to Proton variant of Geely's car, the way to install apps on Geely (GKUI) might be different than the one in Proton (ATLAS).

**Comment by rafaelmillahermoza-star** [comment]
![Screenshot_2025-10-21-14-24-58-171_com facebook katana](https://github.com/user-attachments/assets/166a7661-d9cd-48f4-8a65-217d7203af0e)
Hello, I have a 2023 Geely Okavango. According to the image, the system has already been updated in the Philippines. You can see it for my truck model.
